src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -225,6 +228,10 @@ func (cfg *Config) ApplyDefaults() {
 		cfg.CloudConfig = "https://config.getiantem.org/cloud.yaml.gz"
 	}
 
+	if cfg.InstanceId == "" {
+		cfg.InstanceId = hex.EncodeToString(uuid.NodeID())
```

Comment:
NodeID doesn't change on one device, which is suitable for an instance id. It would be better we prefix it with text, but either the role or country will change.